app.js

```diff
@@ -125,7 +131,7 @@ const protoDino = {
   // Create Dino Compare Method 3 for diet
   compareDiet: function (humanDiet) {
     const article = humanDiet === "omnivore" ? "an" : "a";
-    if (humanDiet === this.diet) {
+    if (humanDiet === titleCase(this.diet)) {
       return `You are ${article} ${humanDiet} and ${this.species} was too!`;
     } else {
       return `You are ${article} ${humanDiet}, but ${this.species} was a ${this.diet}.`;
```

Comment:
On line 153, can you use a const variable to hold the index for human such as **const HUMAN_INDEX = 4;** so it will be easier to change in the future?

Reply:
solved in [001c905](https://github.com/leemoispace/UdacityDinosaurs/pull/1/commits/001c905e77f2367b37693a245588b8c33a47ce34)